### PR TITLE
RHDEVDOCS 5732 using the body for CEL event matching in PAC

### PR DESCRIPTION
**Pull request by mramendi**
Version(s):

please cp to pipelines-1.13

Issue:

RHDEVDOCS 5732

Link to docs preview:

https://68394--docspreview.netlify.app/openshift-pipelines/latest/pac/managing-pipeline-runs-pac#creating-pipeline-run-using-pipelines-as-code_managing-pipeline-runs-pac under "Advanced event matching"

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by savitaashture** [comment]
LGTM

**Comment by piyush-garg** [comment]
lgtm

**Comment by mramendi** [comment]
...or maybe not.

**Review by ppitonak — APPROVED** [review]

File: modules/op-creating-pipeline-run-using-pipelines-as-code.adoc [hunk 1]
```diff
@@ -132,6 +132,42 @@ For advanced CEL-based filtering while using {pac}, you can use the following fi
...
+In addition, you can access the full payload as passed by the Git repository provider. Use the `headers` field to access the headers of the payload, for example, `headers['x-github-event']`. Use the `body` field to access the body of the payload, for example, `body.pull_request.state`.
+
+:FeatureName: using the header and body of the payload for CEL-based filtering with {pac}
+include::snippets/technology-preview.adoc[]
+
+In the following example, the pipeline run starts only if all of the following conditions are true:
+
+* The pull request is targeting the `main` branch
+* The author of the pull request is `superuser`
+* The action is `synchronize`; this action triggers when an update occurs on a pull request
```
Comment (mburke5678) [inline modules/op-creating-pipeline-run-using-pipelines-as-code.adoc:144]: Per the ISG, periods at the end of full sentences in a list. All three need a period.

**Comment by mburke5678** [comment]
@mramendi One nit Otherwise LGTM

**Pull request merged**